packages/stark-ui/assets/styles/_base.scss

```diff
@@ -26,6 +26,12 @@
   width: 100%;
 }
 
+.stark-app-logout-button {
```

Comment:
Well there is a PR regarding the header, when this one will be merged, this piece of code will need to be removed.
Can you add a comment so we don't forget to remove this style?